omni-relayer/src/workers/mod.rs

```diff
@@ -321,8 +335,60 @@ pub async fn process_events(
                                         &config,
                                         &mut redis_connection_manager,
                                         utils::redis::FEE_MAPPING,
-                                        serde_json::to_string(&transfer_message.get_transfer_id())
-                                            .unwrap_or_default(),
+                                        serde_json::to_string(&transfer_id).unwrap_or_default(),
+                                    )
+                                    .await;
+                                }
+                            }
+                        }
+                    }));
+                } else if let Transfer::Utxo {
```

Comment:
Will this execute considering the previous condition

`if let Transfer::Near { .. } | Transfer::Utxo { .. } = transfer.clone()`